src/components/shared/slider/index.tsx

```diff
@@ -35,11 +36,17 @@ const Slider = <
   minLimit = 0,
   maxLimit = 100,
   tooltipContent,
+  showValue = true,
 }: Props<TFormValues>) => {
+  const [currentValue, setCurrentValue] = useState<string>();
   return (
     <View w="75%">
       <View style={styles.tooltipContainer}>
-        {label && <Text>{label}</Text>}
+        {label && (
+          <Text>
+            {label} {showValue && <Text>{currentValue}</Text>}
+          </Text>
+        )}
```

Comment:
We should allow showing just the currentValue (when no label is passed)
```suggestion
        {(label ?? showValue) && (
          <Text>
            {label}
            {label && showValue && ' '}
            {showValue && currentValue}
          </Text>
        )}
```

Reply:
Done.